Category-Colors/category-colors
# Initialize NextJS

# Description
It's a pretty standard install of NextJS using their npx command `npx create-next-app@latest`. 

It includes: 
- New experimental routing system -- Next just launched a new routing mechanism. I figured we could try it out. We can always remove it if we dont like it tho.
- Configured Eslint -- It will help us keep our code bug free.
- Import alias to `@component/`-- It creates an alias to make it easier to import components. Instead of having to use the relative path of a component we can now use `@component/<componentName>`

# QA
After following the instructions in the updated README.md file, go to `http://localhost:3000/` and you should see the standard NextJS hello world page. 
<img width="1281" alt="Screen Shot 2023-03-29 at 6 06 19 PM" src="https://user-images.githubusercontent.com/2035194/228678367-1f48a860-cddc-4da2-a536-c2b8fe3531f9.png">
